front/src/styles/scss/common/components/_typeAndPath.scss

```diff
@@ -29,7 +29,10 @@
       transform: rotate(-40deg);
       transform-origin: left;
       border-radius: var(--border-radius);
+      max-width: 7rem;
       white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
```

Comment:
I don't have any preference, it seems to me better if we can see all/most of the properties of style in one location, this is why I didn't use text-truncate, but I can use it if you prefer

Reply:
For this specific need, it would be only 2 lines (overflow & text-overflow), `max-width` will always be needed, I think it's not mandatory for now. It's approved, you can merge, thx @SharglutDev anyway ;)